### ISSUE TITLE
训练的时候总是使用cpu 显卡驱动和cuda、cudnn已经装好了

### ISSUE THREAD

**Issue by BoldArro1** [MaybeShewill-CV/lanenet-lane-detection#193]
python tools/train_lanenet.py --net vgg --dataset_dir data/training_data_example/
W0331 23:31:55.911763 16288 warnings.py:99] /home/cy/.local/lib/python3.6/site-packages/tensorflow/python/ops/gradients_impl.py:108: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "

2019-03-31 23:31:57.591342: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
I0331 23:31:57.958482 16288 train_lanenet.py:160] Global configuration is as follows:
I0331 23:31:57.958586 16288 train_lanenet.py:161] {'TRAIN': {'EPOCHS': 200010, 'DISPLAY_STEP': 1, 'TEST_DISPLAY_STEP': 1000, 'MOMENTUM': 0.9, 'LEARNING_RATE': 0.0005, 'GPU_MEMORY_FRACTION': 0.85, 'TF_ALLOW_GROWTH': True, 'BATCH_SIZE': 8, 'VAL_BATCH_SIZE': 8, 'LR_DECAY_STEPS': 410000, 'LR_DECAY_RATE': 0.1, 'CLASSES_NUMS': 2, 'IMG_HEIGHT': 256, 'IMG_WIDTH': 512}, 'TEST': {'GPU_MEMORY_FRACTION': 0.8, 'TF_ALLOW_GROWTH': True, 'BATCH_SIZE': 8}}
I0331 23:31:58.959445 16288 train_lanenet.py:169] Training from scratch
I0331 23:32:23.069240 16288 train_lanenet.py:298] Epoch: 1 total_loss= 37.728458 binary_seg_loss= 37.982536 instance_seg_loss= 35.170464 accuracy= 0.509244 mean_cost_time= 16.757031s 
I0331 23:32:23.069416 16288 train_lanenet.py:306] Epoch_Val: 1 total_loss= 1153.302490 binary_seg_loss= 670.360535 instance_seg_loss= 1633.940430 accuracy= 0.455017 mean_cost_time= 4.380603s 
I0331 23:32:43.276617 16288 train_lanenet.py:298] Epoch: 2 total_loss= 36.689880 binary_seg_loss= 35.893253 instance_seg_loss= 35.182602 accuracy= 0.449821 mean_cost_time= 15.123617s

**Comment by MaybeShewill-CV** [comment]
@BoldArro1 查看你的tensorflow gpu安装是否正确

**Comment by BoldArro1** [comment]
重新装了gpu版本的，把cpu的卸载了就可以正常用
 谢谢
